resources/com/griddynamics/devops/mpl/modules/Build/Build.groovy

```diff
@@ -2,7 +2,11 @@
  * Common build module
  */
 
-MPLModule('Maven Build', CFG)
+if ( fileExists('build.gradle')) {
```

Comment:
You changing the interface here, there should be other way because default build system is maven, but still way to use gradle build for circleci automation (also need to be added as a separated pipeline).